test/test_secret_keeper.py

```diff
@@ -1,7 +1,14 @@
 import os
+import random
+import string
 import unittest
+try:
+    from StringIO import StringIO   # for Python2 support
+except ImportError:
+    from io import StringIO
 
-from ridi.secret_keeper import tell, tell_safe, ENVNAME_AWS_ACCESS_KEY, ENVNAME_AWS_SECRET_KEY, ENVNAME_AWS_REGION
+from ridi.secret_keeper import tell, tell_safe
+from ridi.secret_keeper.cmdline import run
 
 try:
     from unittest import mock
```

Comment:
1. Yes, this is for python2.7 support.
https://github.com/ridi/secret-keeper-python/commit/636ef703d49d1940b17a2485f182b5fda07d2276

2. Because it is `boto3`'s `SSM.client.get_parameter` method specification.
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ssm.html#SSM.Client.get_parameter
Since `FakeClient` mocks `SSM.client` class, its method should have the same signature as the original.